packages/checkbox/test/typings/checkbox.types.ts

```diff
@@ -21,11 +25,15 @@ assertType<string>(checkbox.name);
 assertType<string>(checkbox.value);
 
 // Mixins
-assertType<ActiveMixin>(checkbox);
-assertType<ElementMixin>(checkbox);
-assertType<CheckedMixin>(checkbox);
-assertType<DelegateFocusMixin>(checkbox);
-assertType<SlotLabelMixin>(checkbox);
+assertType<ActiveHost>(checkbox);
+assertType<DisabledHost>(checkbox);
```

Comment:
`DisabledHost` and `KeyboardHost ` are no longer available through `ActiveHost` so we have to import them explicitly and add separate assertions. All the complex mixins (`FieldMixin` etc) work the same.

The main change in this PR is to use classes instead of interfaces to have protected methods, and the classes can't have multiple inheritance for obvious reasons. Unfortunately, it forces us to repeat ourselves.